protected/humhub/modules/ui/Module.php

```diff
@@ -41,13 +40,19 @@ class Module extends \humhub\components\Module
         'stream' => 'bars'
     ];
 
+    /**
+     * @var int collapsed post block height
+     * @since 1.15
+     */
+    public int $collapsedPostHeight = 300;
```

Comment:
Can you please add this to the Module class of the Post module. Since this settings is only used by the Post module.

If we would have a global default for all collapse content then we could add it here.

Reply:
Done